translations/base-fr.yaml

```diff
@@ -587,7 +648,9 @@ storyRewards:
 
     reward_miner_chainable:
         title: Extracteur en série
-        desc: Vous avez débloqué <strong>l’extracteur en série</strong> ! Il permet de <strong>transférer ses ressources</strong> à d’autres extracteurs pour augmenter le débit sortant !
+        desc: >-
+            Vous avez débloqué <strong>l’extracteur en série</strong> ! Il permet de <strong>transférer ses ressources</strong> à d’autres extracteurs pour augmenter le débit sortant !<br><br>
+            Note : L'ancien extracteur a été remplacé dans votre barre d'outils maintenant !
```

Comment:
For consistency with the rest of the file please use ` ` U+00A0 NO-BREAK SPACE before `:`, ` ` U+202F NARROW NO-BREAK SPACE before `!`, and `’` U+2019 RIGHT SINGLE QUOTATION MARK. (Pauvres mouches…)